### PR TITLE
Visual differences reuse comment

### LINKED ISSUE #165: Enhance visual tests to update same comment in PR
Repeated runs of the visual tests generate a lot of comments in the PR. This task is logged to just update the same comment with the latest results.

### PR DESCRIPTION
**Pull request by dnbute**
Please always provide the [GitHub issue(s)](../issues) your PR is for, as well as test URLs where your change can be observed (before and after):

Fixes #165 

Test URLs:
- Original: https://www.sunstar.com/<path>
- Before: https://main--sunstar--hlxsites.hlx.live/<path>
- After: https://<branch>--sunstar--hlxsites.hlx.live/<path>

- [ ]  **If you are adding a new block or a variation to an existing block**, please fill below:

  Block library path: https://<branch>--sunstar--hlxsites.hlx.page/tools/sidekick/library.html?plugin=blocks&path=/sidekick/blocks/<your-block>&index=0

### CODE REVIEW

File: .github/workflows/visual-tests.yaml [hunk 1]
```diff
@@ -42,12 +42,27 @@ jobs:
           path: .github/visual-tests/screenshots/*
           name: screenshots
 
+      - name: Find Comment
```
Comment (dnbute) [inline .github/workflows/visual-tests.yaml:45]: test

File: .github/workflows/visual-tests.yaml [hunk 2]
```diff
@@ -42,12 +42,27 @@ jobs:
...
         uses: peter-evans/create-or-update-comment@v3
         with:
           token: ${{ secrets.GITHUB_TOKEN }}
           issue-number: ${{ github.event.number }}
           body: ${{ env.SUMMARY }}
-#        edit-mode: replace
-#        comment-id: ${{ steps.find-comment.outputs.comment-id }}
\ No newline at end of file
+         
+      - name: Update comment
```
Comment (dnbute) [inline .github/workflows/visual-tests.yaml:61]: test2

**Review by sdmcraft — APPROVED** [review]

**Comment by sdmcraft** [comment]
> I could also delete the comment and create a new one for each commit, so the visual differences are always at the bottom, wdyt?

Yes that sounds even better.

**Comment by dnbute** [comment]
@sdmcraft I think this won't work for already opened PRs since they have lots of comments from the bot and I'm not sure how the find comment step will react; But I guess we can manually delete some comments

**Pull request merged**